app/assets/javascripts/discourse/tests/unit/lib/lightbox/process-html-test.js

```diff
@@ -82,7 +82,7 @@ module("Unit | lib | Experimental lightbox | processHTML()", function (hooks) {
     assert.strictEqual(items.length, 3);
   });
 
-  test("fallsback to src when no href is defined for fullsizeURL", async function (assert) {
+  test("fallbacks to src when no href is defined for fullsizeURL", async function (assert) {
```

Comment:
```suggestion
  test("falls back to src when no href is defined for fullsizeURL", async function (assert) {
```

Maybe? "Fallbacks" isn't really a verb :-)